src/ledger/test/InMemoryLedgerTxn.h

```diff
@@ -44,6 +44,12 @@ class InMemoryLedgerTxn : public LedgerTxn
     Database& mDb;
     std::unique_ptr<soci::transaction> mTransaction;
 
+    // For some tests, we need to bypass ledger close and commit directly to the
+    // in-memory ltx. However, we still want to test SQL backed offers. The
+    // "never" committing root sets this flag to true such that offer-related
```

Comment:
I don't know what "this flag" means in the context of this comment -- the config flag saying it's in-memory mode?

Reply:
Ahh, this comment is stale from an earlier version I had where we had a flag and only conditionally wrote to SQL, I'll update it.